docs/root/configuration/http_filters/router_filter.rst

```diff
@@ -8,27 +8,8 @@ that Envoy is deployed for. The filter's main job is to follow the instructions
 configured :ref:`route table <config_http_conn_man_route_table>`. In addition to forwarding and
 redirection, the filter also handles retry, statistics, etc.
 
-.. code-block:: json
-
-  {
-    "name": "router",
-    "config": {
-      "dynamic_stats": "...",
-      "start_child_span": "..."
-    }
-  }
-
-dynamic_stats
-  *(optional, boolean)* Whether the router generates :ref:`dynamic cluster statistics
-  <config_cluster_manager_cluster_stats_dynamic_http>`. Defaults to *true*. Can be disabled in high
-  performance scenarios.
-
-.. _config_http_filters_router_start_child_span:
-
-start_child_span
-  *(optional, boolean)* Whether to start a child :ref:`tracing <arch_overview_tracing>` span for
-  egress routed calls. This can be useful in scenarios where other filters (auth, ratelimit, etc.)
-  make outbound calls and have child spans rooted at the same ingress parent. Defaults to *false*.
+* :ref:`V1 API configuration <config_http_filters_router_v1>`.
+* :ref:`V2 API configuration <envoy_api_msg_filter.http.Router>`.
```

Comment:
is this the procedure to follow for all docs?

Reply:
Yep.